docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md

```diff
@@ -153,146 +153,9 @@ Specifies a list of one or more files names (separated by commas) to be loaded.
 
 A [PCRE2](https://www.pcre.org/current/doc/html/)-based regular expression pattern string, enclosed in single quotes, specifying the file names to match. Click [here](#loading-data-with-pattern-matching) to see an example. For PCRE2 syntax, see http://www.pcre.org/current/doc/html/pcre2syntax.html. 
 
-### formatTypeOptions
+### FILE_FORMAT
```

Comment:
We should keep these content in the `docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md`, why move it out?

Reply:
snow example: https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html#format-type-options-formattypeoptions

Reply:
The reason to move it out is FILE_FORMAT is now a common parameter and is shared by the streaming load API as well. 
https://github.com/datafuselabs/databend/pull/9255#issuecomment-1352541314

Reply:
![Feishu20221221-091917](https://user-images.githubusercontent.com/106025534/208926747-17cb9336-f924-4ffc-962b-df0dcb727f2e.png)

As discussed with @BohuTANG , the new page is to be moved here.